test/missing.jl

```diff
@@ -81,7 +81,7 @@ end
     arithmetic_operators = [+, -, *, /, ^, Base.div, Base.mod, Base.fld, Base.rem]
 
     # All unary operators return missing when evaluating missing
-    for f in [!, ~, +, -]
+    for f in [!, ~, +, -, inv]
```

Comment:
Can you put with with elementary functions instead?

Also, please add an entry in NEWS.md.

Reply:
By elementary functions, do you mean `show`, `convert`, etc.?
And about the compat annotaion, Should I just add `inv` under unary operators in `missing.md`?

Reply:
Sorry, I'm not sure what you mean. In NEWS.md, please add an entry under "New language features".

Reply:
Please check the most recent commit. My query was regarding `needs compat annotation` label.

Reply:
> My query was regarding `needs compat annotation` label.

Ah, OK. I don't think it's appropriate since there's no docstring. @ararslan?

Reply:
Yes, it should be in the existing docstring for `inv`